### PR TITLE
Banned command

### PR DESCRIPTION
**Pull request by ravstar52**
Addresses #11

### CODE REVIEW

File: app/custom_commands.rb [hunk 1]
```diff
@@ -2,9 +2,20 @@
...
+      next unless server_role.name == role
+      has_role = true
+      break
+    end
+    return has_role
+  end
 
   memes = {
     ayy: "https://cdn.drawception.com/images/panels/2015/6-16/xQfqepw73p-2.png",
+	banned: "http://i.imgur.com/O3DHIA5.gif",
```
Comment (laneatomic) [inline app/custom_commands.rb]: whitespace. This should line up with `ayy`

File: app/custom_commands.rb [hunk 2]
```diff
@@ -27,7 +38,7 @@ module CustomCommands
   }
 
   command :meme, description: "Full of dank.", bucket: :memes, usage: "meme [<meme name>]\nAvailable memes:\n- #{memes.keys.join("\n- ")}", min_args: 0, max_args: 1 do |event, meme|
-    break unless event.channel.name.downcase == "bot-abuse"
+    break unless event.channel.name.downcase == "bot-abuse" || role?(event, 'botadmin')
```
Comment (laneatomic) [inline app/custom_commands.rb]: no. I don't want botadmin spamming memes outside of bot-abuse.

File: app/lists.rb [hunk 3]
```diff
@@ -21,22 +21,22 @@ def self.role?(event, role)
       if wing_list.empty?
```
Comment (laneatomic) [inline app/lists.rb]: these changes should not be in this branch. Something weird happened here.

Comment (ravstar52) [inline app/lists.rb]: Nuke the branch and start again?

**Comment by ravstar52** [comment]
Nuke this one, made a new one that *should* be fixed better.

**Pull request closed without merging**